src/c20_server/flask_server.py

```diff
@@ -46,6 +46,14 @@ def _return_result():
 
         return {}, 200
 
+    @app.route('/return_file', methods=['POST'])
+    def _return_file():
+        print('Receiving File from Client ...\n')
```

Comment:
The logging has been added to the server. So this print statement should be changed to use the logging feature.